xrpl/transaction/ledger.py

```diff
@@ -1,27 +1,51 @@
 """High-level methods that fetch transaction information from the XRP Ledger."""
 
-from typing import Any, Dict, cast
+from typing import Any, Dict, Optional, cast
 
 from xrpl.clients import Client, XRPLRequestFailureException
 from xrpl.models.requests import Tx
 from xrpl.models.response import Response
 
 
-def get_transaction_from_hash(tx_hash: str, client: Client) -> Response:
+def get_transaction_from_hash(
+    tx_hash: str,
+    client: Client,
+    binary: bool = False,
+    min_ledger: Optional[int] = None,
+    max_ledger: Optional[int] = None,
+) -> Response:
     """
     Given a transaction hash, fetch the corresponding transaction from the ledger.
 
     Args:
         tx_hash: the transaction hash.
         client: the network client used to communicate with a rippled node.
+        binary: (Optional) If true, return transaction data and metadata as binary
```

Comment:
I don't think there's any need to mark it as optional because the docs will show the type of the param.

Reply:
I removed `(Optional)`